packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1044,12 +1010,18 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     /**
      * Notifies this object to take the snapshot of the container.
+     * @deprecated - Use summarize to get summary of the container runtime.
      * @param tagMessage - Message to supply to storage service for writing the snapshot.
      */
     public async snapshot(tagMessage: string, fullTree: boolean = false): Promise<ITree> {
```

Comment:
Should fullTree argument be removed given it's not used?

Reply:
It's used only for sorting question, which does not look right, given that we pass true to all the interesting function (generation of actual snapshot) - we should just remove it and always sort.

Reply:
Removed it.

Reply:
Also removed tagMessage since it was unused.

Reply:
@vladsud  Do we still need to sort it after I changed the comparison in replay tool to deep compare the objects? The snapshot tests pass if I remove the sort.